libs/agno/agno/memory/manager.py

```diff
@@ -712,6 +717,166 @@ def _get_first_n_memories(self, user_id: str, limit: Optional[int] = None) -> Li
 
         return sorted_memories_list
 
+    def optimize_memories(
+        self,
+        user_id: Optional[str] = None,
+        strategy: Union[
+            MemoryOptimizationStrategyType, MemoryOptimizationStrategy
+        ] = MemoryOptimizationStrategyType.SUMMARIZE,
+        apply: bool = False,
```

Comment:
I think this should be true by default. That seems like the default behaviour and that a "dry run" should be opt-in?